### PR TITLE
Add more audio knobs (ptime, bitrate, vad)

### PR DESCRIPTION
**Pull request by juberti**
@samdutton @phoglund PTAL. Moved out the SDP parsing stuff to its own file since keeping a local copy was getting too nasty (sorry Patrik for the flip-flop on this)

@fippo you will probably find this useful as well

### CODE REVIEW

**Comment by samdutton** [comment]
LGTM – sensible to create sdputils.js.

**Comment by phoglund** [comment]
Sorry, I missed this.

I don't quite remember the discussion, but I did copy those SDP routines from AppRTC js earlier right? At least now we can't use the same file in both since the samples and apprtc repos have split.

I am in favor of moving those routines out to their own file anyway, so these changes lgtm.

**Comment by fippo** [comment]
@juberti ping -- i'd like to reference this demo with VAD settings... :-)

**Comment by KaptenJansson** [comment]
Ping

**Comment by KaptenJansson** [comment]
@juberti PTAL

**Comment by KaptenJansson** [comment]
FYI This needs to be re-based before a merge.

File: src/content/peerconnection/audio/index.html [hunk 1]
```diff
@@ -45,8 +45,9 @@
       </div>
     </div>
 
-    <div id="buttons">
-      <select id="codec">
+    <div id="params">
+       Codec:
```
Comment (samdutton) [inline src/content/peerconnection/audio/index.html]: These labels should be in label elements. (See [constraints example](http://webrtc.github.io/samples/src/content/peerconnection/constraints/).)

**Comment by samdutton** [comment]
LGTM apart from label element nitpick.

**Comment by KaptenJansson** [comment]
Ping, would be real nice to get this in but it needs a merge with master and it's nits fixed.

**Comment by samdutton** [comment]
Label element nitpick could be moved to an issue if you want to merge.

**Comment by KaptenJansson** [comment]
Merge conflict still needs to be resolved, might as well fix the labels then.

**Pull request closed without merging**